O/OpenBLAS/common.jl

```diff
@@ -22,10 +22,11 @@ function openblas_sources(version::VersionNumber; kwargs...)
     ]
 end
 
-function openblas_script(;num_64bit_threads=32, kwargs...)
+function openblas_script(;num_64bit_threads=32, openblas32=0, kwargs...)
```

Comment:
Can `openblas32` be a `Bool`?  Personally, I'm always confused by the meaning of 0 and 1 in terms of logical values, I'd find `true` and `false` clearer.